### DISCUSSION TITLE
VCP (aka MCCS) version for display is undetected or less than 2.0. Output may not be accurate

### DISCUSSION THREAD

**Discussion by gadefox** [rockowitz/ddcutil#342] [Q&A]
I'm using an old laptop w/ `nvidia` legacy driver. Any idea why this message is in `.xsession-errors`? Thanks..

**Comment by rockowitz** [comment]
Just saw this question in Discussions.  I seem to recall having addressed it but apparently not.

If you're asking the reason for the error, reading VCP feature xDF probably failed.    If the problem is intermittent, the ***--stats*** and ***--ddc*** options should give some clue as to what is going on; try increasing the sleep multiplier. If the problem always occurs, does the monitor implement features xDF?

If you're asking why the error appears in .xsession-errors I suppose it has something to do with how messages are routed in your environment.  **ddcutil** is writing the message to stdout.
